.gx/lastpubver

```diff
@@ -1 +1 @@
-0.4.18: QmUJYo4etAQqFfSS2rarFAE97eNGB8ej64YkRT2SmsYD4r
+0.4.20: QmNVba6esWZegye6kfURL5YnQapDYqfiemCUGhstMSFTpj
```

Comment:
go-ipfs releases are a bit more involved as this is a top-level package, you shouldn't call `go release` here.

Reply:
Thanks, still new to gx- as you can probably tell with how messy things are!